MarvelApiClient/DefaultHeadersRequestInterceptor.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DefaultHeadersRequestInterceptor.swift
+//  MarvelAPIClient
+//
+//  Created by Pedro Vicente Gomez on 04/01/16.
+//  Copyright © 2016 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamNetworking
+
+class DefaultHeadersRequestInterceptor: BothamRequestInterceptor {
```

Comment:
Interceptor mechanism, that to set Headers you have yo implement an interceptor,

Maybe if we see that a recuring feature we can define DefaultHeadersRequestInterceptor as part of Botham, and have as param or method of HTTPClient.